ledger-core/v2/conveyor/smachine/state_updates.go

```diff
@@ -284,39 +288,39 @@ func init() {
 				// TODO work in progress
 				panic("work in progress")
 
-				//switch isValid, isBusy := waitOn.getIsValidAndBusy(); {
```

Comment:
This is an incomplete section, hence it is commented out, but I'll get back to it